conda.recipe/meta.yaml

```diff
@@ -31,14 +31,14 @@ requirements:
     - versioningit
 
   build:
-    - mantid
+    - mr_reduction
     - matplotlib
```

Comment:
`mr_reduction` requires `mantidworkbench` which in turn requires `matplotlib` and `qtpy`. Thus remove these

Reply:
Thanks for catching that! By the way, I will rebase when Maria's PR #152 gets merged, since it adds a package test to PR workflows.